.github/workflows/main.yml

```
@@ -0,0 +1,52 @@
+name: collect logs from runner
```

This workflow does not collect logs from runner. It collects logs from workflows in the tarantool/tarantool repo. And the purpose of this is to update flaky test stats. 

```suggestion
name: Update test stats
```